src/WebJobs.Script.Grpc/Server/GrpcServer.cs

```diff
@@ -23,34 +24,31 @@ public GrpcServer(FunctionRpc.FunctionRpcBase serviceImpl)
             };
         }
 
-        public Task StartAsync() {
+        public Uri Uri => new Uri($"http://127.0.0.1:{_server.Ports.First().BoundPort}");
+
+        public Task StartAsync()
+        {
             _server.Start();
             return Task.CompletedTask;
         }
 
         public Task ShutdownAsync() => _server.ShutdownAsync();
 
-        public Uri Uri => new Uri($"http://127.0.0.1:{_server.Ports.First().BoundPort}");
-
-        #region IDisposable Support
-        private bool disposedValue = false; // To detect redundant calls
-
         protected virtual void Dispose(bool disposing)
         {
-            if (!disposedValue)
+            if (!_disposed)
             {
                 if (disposing)
                 {
                     _server.ShutdownAsync();
                 }
-                disposedValue = true;
+                _disposed = true;
             }
         }
 
         public void Dispose()
         {
             Dispose(true);
```

Comment:
The old fxcop rules we had would also require a `GC.SuppressFinalize(this)` call. We're using different rules now?